mrp_production_line_group/wizard/change_production_qty.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models, api, _
+from odoo.exceptions import UserError
+
+
+class ChangeProductionQty(models.TransientModel):
+    _inherit = 'change.production.qty'
+
+    @api.multi
+    def change_prod_qty(self):
+        for wizard in self:
+            production = wizard.mo_id
+            if production.lines_grouped:
+                raise UserError(_(
+                    'This MO has been grouped, so change product qty is no more'
```

Comment:
I propose to insert production number in error message

Reply:
done!